swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -29,16 +32,28 @@ enum LogLevel {
     case assert, error, warning, info, debug
 }
 
+/// For the hardcoded values that represent the privacy settings we have an example of more enum utilization here.
 enum PrivacySetting: String {
     case exclude = "fs-exclude"
     case mask = "fs-mask"
     case unmask = "fs-unmask"
 }
 
+/**
+ This is a convenience method that takes a case and a dictionary to title the event and provide properties.
+    - Parameter event: `browsing`, `itemSelected`, `viewCart`, `addToCart`, `checkout`, `removeFromCart`, and `crash` (This could be different for you based on the flow of your app.).
```

Comment:
You've declared the structure above for the `Event` enum.  You shouldn't need to explain it again here as a developer I'd simply go look at. what an `Event` enum is.  Consider this as a maintenance chore.  If the enum changes, now you're going back to all your comments to update them.  Instead, says something like, "The `Event` enum defines the value for a FullStory event name"